Source/SpatialGDK/Public/Utils/SchemaOption.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include <improbable/c_worker.h>
+#include "UniquePtr.h"
+
+template <typename T>
+class TSchemaOption
+{
+public:
+	TSchemaOption() = default;
+	~TSchemaOption() = default;
+
+	TSchemaOption(const T& InValue)
+	{
+		Value = MakeUnique<T>(InValue);
```

Comment:
Can we do this in the initializer list?
`TSchemaOption(const T& InValue)
    : Value(MakeUnique<T>(InValue))`

Reply:
Yeah potentially. I just didn't see `TOptional` or `worker::Option` do that so I refrained.